### PR TITLE
Custom token program (BTCi mint)

### PR DESCRIPTION
**Pull request by sviatoslav-alekseev**
#### Problem
We should be able to mint the BTCi token.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,11 @@
 target
+target_0
```
Comment (dragvs) [inline .gitignore:2]: I'd do some kind of wildcard already here

File: token/cli/src/main.rs [hunk 2]
```diff
@@ -242,23 +242,6 @@ fn is_multisig_minimum_signers(string: String) -> Result<(), String> {
     }
 }
 
-fn is_valid_token_program_id<T>(string: T) -> Result<(), String>
```
Comment (dragvs) [inline token/cli/src/main.rs:245]: Should we remove it completely?

**Review by dragvs — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (sviatoslav-alekseev) [reply .gitignore]: Done

File: token/cli/src/main.rs (hunk 2 above)
Comment (sviatoslav-alekseev) [reply token/cli/src/main.rs]: Reverted this change

**Review by dragvs — APPROVED** [review]

**Pull request merged**